rust-utils/package.json

```diff
@@ -10,7 +10,7 @@
   "license": "MIT",
   "devDependencies": {
     "@napi-rs/cli": "^2.16.5",
-
+    "rimraf": "5.0.5",
```

Comment:
We've defined rimraf in the root package.json, so you should be able to call it from there. In other words, there's no need to add it here.